frontend_tests/node_tests/pm_list.js

```diff
@@ -48,8 +51,28 @@ global.people.add_in_realm(bob);
 global.people.add_in_realm(me);
 global.people.initialize_current_user(me.user_id);
 
+(function test_get_conversation_li() {
+    var test_conversation = 'foo@example.com,bar@example.com';
+    var error_msg;
+    global.blueslip.warn = function (error) {
+        error_msg = error;
+    };
+    pm_list.get_conversation_li(test_conversation);
+    assert.equal(error_msg, 'Unknown conversation: ' + test_conversation);
+}());
+
+(function test_close() {
+  var collapsed;
```

Comment:
ohh missed it. Going trough indentation again.
On a side note don't have a linter for this thing?